### ISSUE TITLE
Containers don't respawn loot inside of them overtime

### ISSUE THREAD

**Issue by Nikolassparrow** [QuentinGruber/h1z1-server#1932]
**Describe the bug**
Was a bug caught by Avcio and I while talking about loot tables. Essentially, normal world spawns on H1EMU will spawn items over each pass-through in lootSpawning. This happens because on the call to spawn world loot, the function takes in all the positions of every possible spawn in the world, and whatever chance that item gets to spawn, will spawn in any available position based upon the previous spawns.

For example: lets say there's a room with 5 spawns, and 2/5 spots are taken on the 1st call. On the 2nd call, perhaps another spot is taken for 3/5. Keep repeating this process for the amount of positions in a given area.

TLDR: Normal world spawns happen overtime because vacant spots from the previous respawn are filled in.

**To Reproduce**
Revert #1907, load onto a world, find any container and observe overtime the amount of items that fill up that specific container.

**Expected behavior**
There should be few items that spawn in containers initially, but overtime fill up as time goes by until it reaches the maxItems inside of said container.

**Screenshots**
https://streamable.com/8gayr4

**Additional context**
#1907 will probably need to be reverted, and #1930 to be adjusted based upon this issue
